components/ScrollMonitor.tsx

```diff
@@ -27,7 +27,7 @@ const useStyles = makeStyles(
   })
 );
 
-const ScrollMonitor = ({ className: classNameProp, progress }: Props, ref: LegacyRef<HTMLDivElement>): ReactElement => {
+const ScrollMonitor = ({ className: classNameProp }: Props, ref: LegacyRef<HTMLDivElement>): ReactElement => {
```

Comment:
When I updated ESlint, I got an error noting that `progress` was unused so I fixed that issue here to get linting to pass.